web/modules/custom/ai_screening_project_track/src/Form/ThresholdsForm.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\ai_screening_project_track\Form;
+
+use Drupal\Core\DependencyInjection\AutowireTrait;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\State\StateInterface;
+use Drupal\ai_screening_project_track\Helper\ProjectTrackTypeHelper;
+
+/**
+ * Configure AI Screening project track settings for this site.
+ */
+final class ThresholdsForm extends FormBase {
+  use AutowireTrait;
+
+  public function __construct(
+    private readonly ProjectTrackTypeHelper $projectTrackTypeHelper,
+    private readonly StateInterface $state,
+  ) {
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'ai_screening_project_track_thresholds';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $projectTrackTypes = $this->projectTrackTypeHelper->loadTerms();
+    foreach ($projectTrackTypes as $termId => $projectTrackType) {
+      $projectTrackConfiguration = $this->projectTrackTypeHelper->getConfiguration($projectTrackType);
+
+      $form['project_track'][$termId] = [
+        '#type' => 'fieldset',
+        '#title' => $projectTrackType->label(),
+      ];
+
+      $form['project_track'][$termId]['term_wrapper'] = [
+        '#type' => 'container',
+      ];
+
+      $form['project_track'][$termId]['term_wrapper']['text'] = [
+        '#markup' => '<div class="mb-3 font-bold">' . $projectTrackType->getDescription() . '</div>',
+      ];
+
+      foreach ($projectTrackConfiguration['dimensions'] as $key => $dimension) {
```

Comment:
```suggestion
      foreach ($projectTrackConfiguration[ProjectTrackTypeHelper::CONFIGURATION_KEY_DIMENSIONS] as $key => $dimension) {
```

You can also use `ProjectTrackTypeHelper->getDimensions` (cf. https://github.com/itk-dev/ai-screening/blob/develop/web/modules/custom/ai_screening_project_track/src/Helper/ProjectTrackTypeHelper.php#L90)